lib/adaptivequemgr.go

```diff
@@ -117,6 +118,19 @@ type BindCount struct {
 	Workers map[string]*WorkerClient // lookup by ticket
 }
 
+func bindEvictNameOk(bindName string) (bool) {
+	if len(GetConfig().BindEvictionNames) == 0 {
+		// for tests, allow all names to be subject to bind eviction
+		return true
+	}
+	for _, okSubname := range strings.Split(GetConfig().BindEvictionNames,",") {
+		if strings.Contains(bindName, okSubname) {
```

Comment:
What if there are multiple matches ? for ex: account_number and account_number_two are part of the where clause?

Reply:
should we also do case insensitive comparison?

Reply:
Should we also add a warning message when the length of bind name is less than 7?

Reply:
For multiple binds (account_number, account_number_two), both get evaluated for eviction. First one that matches goes through.

I've added case insensitive.

For short bind names, we'd probably log too much.